azure-quantum/tests/unit/common.py

```diff
@@ -261,9 +262,7 @@ def resume_recording(self):
 
     @property
     def is_playback(self):
-        return (self.connection_params.subscription_id == SUBSCRIPTION_ID
-                and not self.in_recording 
-                and not self.is_live)
+        return not self.in_recording and not self.is_live
```

Comment:
I've tried to check everything, and it seems that it should be safe to remove the `subscription_id` from the condition, so we consider it a playback just when the recording file is present.